### DISCUSSION TITLE
4.0.1.0

### DISCUSSION THREAD

**Discussion by danielkerr** [opencart/opencart#11540] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/opencart/opencart/releases/tag/'>4.0.1.0</a>.</em>

**Comment by mhcwebdesign** [comment]
Thanks for this, and all the hard work that went into it.

There are still a number of outstanding issues, hopefully they can be addressed soon.

**Comment by felipoantonoff** [comment]
Congratulations for the work and for everyone who helped or contributed in some way.
